### PR TITLE
Bug/vspc 109

### PR DESCRIPTION
**Pull request by Sagar-k93**
When creating a new link, if no label is given, there should be proper validation that tells the user which field to fill in. For new space links, both the label and linked space is mandatory. For module link, label and linked module are mandatory and for external links, the external link and the label are mandatory fields for the link to be established on the space.

### CODE REVIEW

**Comment by jdamerow** [comment]
Resolve conflicts please

File: vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp [hunk 1]
```diff
@@ -1392,91 +1429,45 @@ $( document ).ready(function() {
...
+	      </div>
+	      
+	      <div class="row">
+	      <div class="col-sm-3" style="padding-right: 0px;">
+		  <label><small>Image:</small> </label>
+		  </div>
+	      <div class="col-sm-9">
+	      <input type="file" class="form-control-xs" type="text" name="externalLinkImage" id="externalLinkImage"><br>
+	      </div>
+	      </div>
```
Comment (prjadhav14) [inline vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp:1466]: Indentation

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
The error alert needs to always be visible. Right now, I scrolled down a little, created a link, clicked "create button" and it seems like nothing happens because the error alert is too far on the top to be visible.

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Pull request merged**